modules/nodes-pods-secrets-creating-web-console-secrets.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/nodes-pods-secrets.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-pods-secrets-creating-web-console-secrets.adoc_{context}"]
+= Create a Secret object by using the web console
+
+As an administrator, you can create secrets using the web console. 
+
+. Navigate to *Workloads* → *Secrets*.
+. Click *Create* → *From YAML*.
+.. Edit the YAML to your specifications. 
```

Comment:
Would it be helpful to insert an alternative sample YAML here for clarity?

Reply:
@mpandya15 Yes. I think it would be helpful to add an example YAML. It would provide consistency with the other sections, each of which have an example.  Maybe list the secret types and point to the following sections?

Reply:
Also, maybe swap your new module with Creating opaque secrets, so that all of the secret type modules are together?

Reply:
@mburke5678 Thanks a lot for your feedback! I added the sample YAML, but is there another custom, Secret YAML example I can use for this, since we already have an example of Opaque? Opaque is the default in the console but maybe something different would be better.